docs/reference/support-duration.md

```diff
@@ -1,9 +1,8 @@
 # Ubuntu Core support duration
 
-The support period for Ubuntu Core follows the [Long Term Support](https://ubuntu.com/about/release-cycle) (LTS) model adopted by all major releases of Ubuntu.
+The support period for Ubuntu Core follows the [Long Term Support](https://ubuntu.com/about/release-cycle) (LTS) model adopted by all major releases of Ubuntu, with 10 years of Ubuntu Core support as standard.
```

Comment:
, with 10 years of standard security maintenance.

Reply:
And I think we should call out to what = Core XX snap and Snapd snap 
Note:
Kernel has to document this for their snaps